portal/config/eproms/ScheduledJob.json

```diff
@@ -77,6 +77,18 @@
       "schedule": "0 0 1 1 0",
       "task": "deactivate_tous"
     },
+    {
+      "active": false,
+      "args": null,
+      "kwargs": {
+        "notification": "website_consent_update",
+        "roles": ["patient", "staff", "staff_admin"]
+      },
+      "name": "Website consent update notification",
+      "resourceType": "ScheduledJob",
+      "schedule": "0 0 1 1 0",
```

Comment:
@pbugni Should we be including a `schedule` if these are manual, one-off jobs (run when something is updated in LR)?